src/components/external-source/ExternalSourcePanelEntry.svelte

```diff
@@ -0,0 +1,140 @@
+  import EyeSlash from 'bootstrap-icons/icons/eye-slash.svg?component';
+  import { get } from 'svelte/store';
+  import { derivationGroupVisibilityMapWritable, externalSources } from '../../stores/external-source';
+  import { plan } from '../../stores/plan';
+  import { plugins } from '../../stores/plugins';
+  import type { User } from '../../types/app';
+  import type { DerivationGroup, ExternalSourceSlim } from '../../types/external-source';
+  import effects from '../../utilities/effects';
+  import { formatDate } from '../../utilities/time';
+  import { tooltip } from '../../utilities/tooltip';
+  import Collapse from '../Collapse.svelte';
+
+  export let derivationGroup: DerivationGroup;
+  export let user: User | null;
+
+  let relevantSources: ExternalSourceSlim[] = [];
+
+  $: enabled = $derivationGroupVisibilityMapWritable[derivationGroup.name] ?? true;
+  $: relevantSources = $externalSources.filter(source => derivationGroup.name === source.pkey.derivation_group_name);
+
+  function onChange() {
+    derivationGroupVisibilityMapWritable.set({
+      ...get(derivationGroupVisibilityMapWritable),
+      [derivationGroup.name]: !enabled,
+    });
+  }
+
+  async function deleteEmptyDerivationGroup() {
+    if (enabled) {
+      // Delete plan derivation group association
+      await effects.deleteDerivationGroupForPlan(derivationGroup.name, $plan, user);
+    }
+    // Delete the derivation group itself
+    await effects.deleteDerivationGroup(derivationGroup.name, user);
+  }
+</script>
+
+<div class="external-source-pairing">
+  <Collapse
+    title={derivationGroup.name}
+    tooltipContent={'Derivation group ' + derivationGroup.name}
+    defaultExpanded={false}
+  >
+    <span slot="right" style:display="flex">
+      <p class="derived-event-text">
+        {derivationGroup.derived_event_total} derived events
+      </p>
+      {#if enabled === true}
+        <button
+          class="st-button icon"
+          on:click|stopPropagation={onChange}
+          use:tooltip={{ content: 'Show in timeline', placement: 'top' }}
+        >
+          <Eye style="color: rgba(143, 143, 143, 1)" />
+        </button>
+      {:else}
+        <button
+          class="st-button icon"
+          on:click|stopPropagation={onChange}
+          use:tooltip={{ content: 'Hide in timeline', placement: 'top' }}
+        >
+          <EyeSlash style="color: rgba(143, 143, 143, 0.5)" />
+        </button>
+      {/if}
+    </span>
+
+    {#if relevantSources.length}
+      {#each relevantSources as source}
+        <!-- Collapsible details -->
+        <Collapse title={source.pkey.key} tooltipContent={source.pkey.key} defaultExpanded={false}>
+          <span slot="right">
+            <p style:color="gray" style:text-wrap="nowrap">
+              {derivationGroup.sources.get(source.pkey.key)?.event_counts} events
+            </p>
+          </span>
+          <p>
+            <strong>Key:</strong>
```

Comment:
Prefer div with `className='st-typography-bold'` with a `className='st-typography-body'` on the parent or somewhere above so that stellar type styles are applied here.